packages/titus-backend/index.js

```diff
@@ -16,7 +16,18 @@ process.on('unhandledRejection', err => {
 
 const main = async () => {
   const server = fastify(config.fastify)
-  server.register(helmet)
+  server.register(require('fastify-swagger'), require('./lib/config/swagger'))
+  server.register(helmet, instance => ({
+    contentSecurityPolicy: {
+      directives: {
+        ...helmet.contentSecurityPolicy.getDefaultDirectives(),
+        'form-action': [`'self'`],
+        'img-src': [`'self'`, 'data:', 'validator.swagger.io'],
+        'script-src': [`'self'`].concat(instance.swaggerCSP.script),
+        'style-src': [`'self'`, 'https:'].concat(instance.swaggerCSP.style)
+      }
+    }
+  }))
```

Comment:
I wonder if, instead of moving swagger here, we shouldn't move the two registrations to lib/server.js instead

Reply:
I considered doing this. It wasn't clear to me whether registering `helmet` as soon as possible was best practice. Someone chose to register it in `index.js`, rather than in `server.js`, so I figured I would defer to their decision to do so. I don't mind moving it, either. I'm simply not familiar enough with the internals of `fastify`, `helmet` or `fastify-helmet` to know whether there's a good reason to register `fastify-helmet` as early as possible.

Reply:
Knowing the codebase I don't see a good reason why it should be done here instead of lib/server.js, I'd move it there. If you then revert the package-lock file changes, this PR is good to go.